llmfoundry/models/llm_embed/modeling_llm_embed.py

```diff
@@ -219,13 +233,13 @@ def format_queries_batch(
         Here ``n`` is the step size, which represents the number of hard
         negatives per passage.
         """
+        assert self.step_size
         queries = {}
+        indices = []
```

Comment:
The default behavior is a bit confusing. Can you add a comment?

Reply:
This is just for pyright.

Reply:
I see. It would be nice if this was tidier I guess.

What about something like:

```
# outside the loop
indices = list(range(0, last_hidden_state.size(0), self.step_size))

for key in batch:
  queries[key] = batch[key][indices, :]
return queries, last_hidden_state[indices, :, :]
```